app/models/nic/interface.rb

```diff
@@ -74,9 +74,11 @@ def identifier_change
       old_value = self.identifier_was || ''
```

Comment:
I can't find a scenario where this would break. Interfaces have empty identifier only when they are built in Foreman and if it causes problems in some cases, user can set identifiers explicitly. So :+1: from me. Test looks good, I'll give it few tests and if everything is alright and no one objects I'll merge.